### PR TITLE
[lambda-calculus/es-es] Translate Lambda Calculus to es_ES

### PR DESCRIPTION
**Pull request by AlburIvan**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: es-es/lambda-calculus-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+## Variables Libres vs. Enlazadas:
+
+- En la función `λx.x`, "x" se llama una variable enlazada porque está tanto en
+ el cuerpo de la función como en el parámetro.
+- En `λx.y`, "y" se llama variable libre porque nunca se declara de antemano.
+
+## Evaluación:
+
+Evaluación se realiza a través de
+[β-Reduction](https://en.wikipedia.org/wiki/Lambda_calculus#Beta_reduction),
```
Comment (Menelion) [inline es-es/lambda-calculus-es.html.markdown]: Es también disponible en español: https://es.wikipedia.org/wiki/C%C3%A1lculo_lambda#%CE%B2-reducci%C3%B3n

File: es-es/lambda-calculus-es.html.markdown [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    - ["Max Sun", "http://github.com/maxsun"]
+    - ["Yan Hui Hang", "http://github.com/yanhh0"]
+translators:
+    - ["Ivan Alburquerque", "https://github.com/AlburIvan"]
+---
+
+# Cálculo Lambda
+
+Cálculo Lambda (Cálculo-λ), originalmente creado por 
+[Alonzo Church](https://en.wikipedia.org/wiki/Alonzo_Church),
```
Comment (Menelion) [inline es-es/lambda-calculus-es.html.markdown]: Hay también un pequeño artículo sobre él en Español: https://es.wikipedia.org/wiki/Alonzo_Church

File: es-es/lambda-calculus-es.html.markdown [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+- `(λx.x)a` evalúa a: `a`
+- `(λx.y)a` evalúa a: `y`
+
+Incluso puedes crear funciones de orden superior:
+
+- `(λx.(λy.x))a` evalúa a: `λy.a`
+
+Aunque el cálculo lambda tradicionalmente solo admite funciones 
+de un solo parámetro, podemos crear funciones multiparamétricas usando 
+una técnica llamada [currying](https://en.wikipedia.org/wiki/Currying).
```
Comment (Menelion) [inline es-es/lambda-calculus-es.html.markdown]: Currificación: https://es.wikipedia.org/wiki/Currificación

File: es-es/lambda-calculus-es.html.markdown [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+`a AND b` es equivalente a: `λab.IF a b F`
+
+`a OR b` es equivalente a: `λab.IF a T b`
+
+`a NOT b` es equivalente a: `λa.IF a F T`
+
+*Note: `IF a b c` es esencialmente diciendo: `IF((a b) c)`*
+
+## Numeros:
```
Comment (Menelion) [inline es-es/lambda-calculus-es.html.markdown]: Números

File: es-es/lambda-calculus-es.html.markdown [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+Para la parte interior `λx.f(f x)`:
+```
+  λx.f(f x)
+= S (λx.f) (λx.(f x))          (case 3)
+= S (K f)  (S (λx.f) (λx.x))   (case 2, 3)
+= S (K f)  (S (K f) I)         (case 2, 1)
+```
+
+Asi que:
```
Comment (Menelion) [inline es-es/lambda-calculus-es.html.markdown]: Así que

File: es-es/lambda-calculus-es.html.markdown [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+I = ιι
+K = ι(ιI) = ι(ι(ιι))
+S = ι(K) = ι(ι(ι(ιι)))
+```
+
+## Para una lectura más avanzada:
+
+1. [A Tutorial Introduction to the Lambda Calculus](http://www.inf.fu-berlin.de/lehre/WS03/alpi/lambda.pdf)
+2. [Cornell CS 312 Recitation 26: The Lambda Calculus](http://www.cs.cornell.edu/courses/cs3110/2008fa/recitations/rec26.html)
+3. [Wikipedia - Lambda Calculus](https://en.wikipedia.org/wiki/Lambda_calculus)
```
Comment (Menelion) [inline es-es/lambda-calculus-es.html.markdown]: https://es.wikipedia.org/wiki/Cálculo_lambda

**Comment by AlburIvan** [comment]
@Menelion should be done now :)

File: es-es/lambda-calculus-es.html.markdown [hunk 7]
```diff
@@ -0,0 +1,215 @@
+---
+category: Algorithms & Data Structures
+name: Lambda Calculus
+contributors:
+    - ["Max Sun", "http://github.com/maxsun"]
+    - ["Yan Hui Hang", "http://github.com/yanhh0"]
+translators:
+    - ["Ivan Alburquerque", "https://github.com/AlburIvan"]
+---
```
Comment (divayprakash) [inline es-es/lambda-calculus-es.html.markdown]: Please add missing lang code.
```
lang: es-es
```

**Comment by Menelion** [comment]
LGTM. @divayprakash, Please approve :).

**Comment by Menelion** [comment]
¡Muchas gracias!

**Pull request merged**